pyniva/get_data.py

```diff
@@ -103,5 +109,33 @@ def token2header(token_file):
     }
 
     token = jwt.encode(payload, cmrsa["private_key"], algorithm="RS256")
-    header = { "Authorization": b"Bearer " + token }
-    return header
\ No newline at end of file
+    header = {"Authorization": b"Bearer " + token}
+    return header
+
+
+def get_newly_inserted_data(start_time: dt.datetime, end_time: dt.datetime, aggregate: bool,
+                            meta_host: str, ts_host: str, headers=None, session=None):
+    params = {'start': start_time.isoformat(),
+              'end': end_time.isoformat(),
+              'aggregate': aggregate}
+
+    rq = session or requests
+    if headers is None:
+        headers = {}
+    trace_id = str(uuid.uuid4())
+    headers['Trace-Id'] = trace_id
```

Comment:
probably not something that we'd want to spend time on now, but all these headers should ideally be added as request middleware made by Johannes https://github.com/NIVANorge/nivacloud-logging#tracing-with-requests